src/Compilers/Core/Portable/Operations/OperationNodes.cs

```diff
@@ -7854,6 +7855,9 @@ public override IEnumerable<IOperation> Children
         /// Constant value of the pattern.
         /// </summary>
         public abstract IOperation Value { get; }
+
+        public ITypeSymbol InputType { get; }
```

Comment:
>InputType [](start = 27, length = 9)

What does the input type represent? Could it be removed?
I'd expect the constant/value to have a type and that's it.
Also, I don't see `InputType` in any of the test outputs.

Reply:
I looked at the spec, but saw not mention of "input type". 
In `e is <pattern>`, isn't the input type just the type of `e`?

---
In reply to: [244056434](https://github.com/dotnet/roslyn/pull/31967#discussion_r244056434) [](ancestors = 244056434,244029510)